prow/tide/tide.go

```diff
@@ -618,7 +618,11 @@ func pickSmallestPassingNumber(log *logrus.Entry, ghc githubClient, prs []PullRe
 // accumulateBatch returns a list of PRs that can be merged after passing batch
 // testing, if any exist. It also returns a list of PRs currently being batch
 // tested.
-func accumulateBatch(presubmits map[int]sets.String, prs []PullRequest, pjs []kube.ProwJob) ([]PullRequest, []PullRequest) {
+func accumulateBatch(presubmits map[int]sets.String, prs []PullRequest, pjs []kube.ProwJob, headContextOnly bool) ([]PullRequest, []PullRequest) {
+	if headContextOnly {
```

Comment:
Passing a bool into a func to do early return seems smelly -- why not conditionally make the call?